test/integration/net_test.go

```diff
@@ -70,6 +70,10 @@ func TestNetworkPlugins(t *testing.T) {
 					t.Skipf("flannel is not yet compatible with Docker driver: iptables v1.8.3 (legacy): Couldn't load target `CNI-x': No such file or directory")
 				}
 
+				if ContainerRuntime() != "docker" && tc.name == "false" {
+					t.Skipf("skipping the test as CNI is required for container runtime %s", ContainerRuntime())
```

Comment:
Here lets check in the test, for Exit Usage if chi=false and container runtime non-docker